.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: java
```

Comment:
Well... there is no ready-to-use github-loves-Jenkins integration like there is for Travis, with a freely usable server farm behind it. That's another organizational bug to address, really - in other projects that were opensourced via github we were forced to use both Travis and Jenkinsfiles (or legacy configs), just to check PRs as basically compilable before accepting them and submitting to stronger CI torture on internal Jenkins farm.